fuzz/fuzz_adt_set.c

```diff
@@ -68,7 +68,7 @@ test_set_operations(uintptr_t limit2)
 			.trial_post = repeat_with_verbose,
 			.env = &env,
 		},
-		.seed = theft_seed_of_time(),
+		.seed = seed,
 		.trials = 100000,
```

Comment:
Out of scope of this PR, but the trial count might be another good CLI argument. At that point, it might also be better to pass in a config struct to the test (with the seed, trial count, and whatever else).

Reply:
I was thinking just the same - but I'd like to be able to specify that in terms of something meaningful for each particular test. Because they have different densities for their search spaces. I feel like such a CLI argument ought to apply equivalently for all tests. Thoughts?

Reply:
They should probably have different defaults, agreed. If I'm overriding the trial count, I'm usually also just running a single test by name, though. Running all of the properties (either from CI or directly) is usually part of a different workflow for me than running a single property with a huge number of trials (possibly fuzzing it for several hours).